src/Payments.Core/Domain/PaymentEvent.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Payments.Core.Domain
+{
+    public class PaymentEvent
+    {
+        public PaymentEvent()
+        {
+            OccuredAt = DateTime.UtcNow;
+        }
+        [Key]
+        public string Transaction_Id { get; set; }

```

Comment:
@jSylvestre We can leave for now but I'll probably want this to match our db camelCase instead of having just a few fields be different